gammapy/modeling/models/spectral.py

```diff
@@ -822,6 +856,11 @@ def pivot_energy(self):
             E_D = E_0 * \exp{cov(\phi_0, \Gamma) / (\phi_0 \Delta \Gamma^2)}
 
         Formula (1) in https://arxiv.org/pdf/0910.4881.pdf
+
+        Returns
+        -------
+        pivot energy : `~astropy.units.Quantity`
+            If no minimum is found, NaN will be returned.
```

Comment:
Does the nan case arise here?

Reply:
yes

Reply:
For the simple power law, this is an analytical result, no?

Reply:
If there are no error, there is no minimum. So a NaN should be returned

Reply:
Yes indeed! that's the behaviour in the present version. 
Thanks Bruno